openmetadata-ui/src/main/resources/ui/src/pages/ClassificationVersionPage/ClassificationVersionPage.test.tsx

```diff
@@ -0,0 +1,184 @@
+  () => {
+    return jest.fn().mockReturnValue(<div>testErrorPlaceHolder</div>);
+  }
+);
+
+jest.mock(
+  '../../components/Entity/EntityVersionTimeLine/EntityVersionTimeLine',
+  () => {
+    return jest.fn().mockReturnValue(<div>testEntityVersionTimeLine</div>);
+  }
+);
+
+jest.mock('../../components/Loader/Loader', () => {
+  return jest.fn().mockReturnValue(<div>Loader component</div>);
+});
+
+jest.mock('../../components/PageLayoutV1/PageLayoutV1', () => ({
+  __esModule: true,
+  default: jest
+    .fn()
+    .mockImplementation(({ children }) => <div>{children}</div>),
+}));
+
+jest.mock('../../components/PermissionProvider/PermissionProvider', () => ({
+  usePermissionProvider: jest.fn().mockImplementation(() => ({
+    getEntityPermissionByFqn: jest
+      .fn()
+      .mockImplementation(() => Promise.resolve(ENTITY_PERMISSIONS)),
+  })),
+}));
+
+jest.mock('../../utils/PermissionsUtils', () => ({
+  checkPermission: jest.fn().mockReturnValue(true),
+  DEFAULT_ENTITY_PERMISSION: {
+    ViewAll: true,
+    ViewBasic: true,
+  },
+}));
+
+jest.mock('../../rest/tagAPI', () => ({
+  getClassificationByName: jest
+    .fn()
+    .mockImplementation(() => Promise.resolve(MOCK_ALL_CLASSIFICATIONS)),
+  getClassificationVersionData: jest
+    .fn()
+    .mockImplementation(() => Promise.resolve(MOCK_ALL_CLASSIFICATIONS)),
+  getClassificationVersionsList: jest
+    .fn()
+    .mockImplementation(() => Promise.resolve(MOCK_ALL_CLASSIFICATIONS)),
+}));
+
+jest.mock('../../utils/RouterUtils', () => ({
+  getClassificationVersionsPath: jest.fn(),
+  getClassificationDetailsPath: jest.fn(),
+}));
+
+jest.mock('../../utils/ToastUtils', () => ({
+  showErrorToast: jest.fn(),
+}));
+
+describe('ClassificationVersionPage component', () => {
+  it('should render Loader', async () => {
+    render(<ClassificationVersionPage />, {
+      wrapper: MemoryRouter,
+    });
+
+    const loader = screen.getByText('Loader component');
+
+    expect(loader).toBeInTheDocument();
+  });
+
+  it('should render EntityVersionTimeLine', async () => {
+    await act(async () => {
+      render(<ClassificationVersionPage />, {
+        wrapper: MemoryRouter,
+      });
+    });
+
+    const versionData = await screen.findByTestId('version-data');
+    const entityVersionTimeLine = await screen.findByText(
+      'testEntityVersionTimeLine'
+    );
+
+    expect(versionData).toBeInTheDocument();
+    expect(entityVersionTimeLine).toBeInTheDocument();
+  });
+
+  it('should renders ErrorPlaceHolder with permission error', async () => {
+    mockedUsePermissionProvider.mockReturnValue({
+      getEntityPermissionByFqn: jest.fn().mockResolvedValue({}),
+    });
+
+    render(<ErrorPlaceHolder type={ERROR_PLACEHOLDER_TYPE.PERMISSION} />, {
+      wrapper: MemoryRouter,
+    });
+
+    const errorPlaceholder = await screen.findByText('testErrorPlaceHolder');
+
+    expect(errorPlaceholder).toBeInTheDocument();
+  });
```

Comment:
@harsh-vador , we need to check if the error placeholder is rendered by the version component or not, here we are just rendering the error placeholder component and checking it.

Can we check if the version component rendered the error placeholder instead?

Reply:
Sure